aptoide-analytics-default-implementation/build.gradle

```diff
@@ -68,7 +68,7 @@ dependencies {
   }
 
   // Facebook
-  implementation "com.facebook.android:facebook-android-sdk:4.17.0"
+  implementation 'com.facebook.android:facebook-android-sdk:4.28.0'
```

Comment:
According to https://developers.facebook.com/docs/android/upgrading-4x , provided in the pull request description, there are no changes needed to 4.29.0. Why didn't we upgrade to 4.29.0 ?

Reply:
It says it needs no changes related directly to the sdk, but since the sdk upgraded their version to support lib to 27.0.1, we'd have to do the same